DLLSocket/DLLSocket.cbp

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Don't add tools inside the root of the repository, add them inside the `tools` folder, unless you like a messy codebase.